SUMMARY.md

```diff
@@ -51,6 +51,8 @@
 * [Bridging to Moonbeam](blockchain-layer-1/bridging-to-moonbeam.md)
 * [Deployed smart contracts](blockchain-layer-1/deployed-smart-contracts.md)
 * [Other blockchains](blockchain-layer-1/other-blockchains.md)
+* [Indexers](indexers/overview.md)
```

Comment:
This should be moved under OriginTrail Parachain section (line 43), as it is indexer only for OriginTrail Parachain.